lms/djangoapps/grades/new/course_grade.py

```diff
@@ -24,8 +27,40 @@ class CourseGrade(object):
     def __init__(self, student, course, course_structure):
         self.student = student
         self.course = course
+        self.course_version = getattr(course, 'course_version', None)
+        self.course_edited_timestamp = getattr(course, 'subtree_edited_on', None)
```

Comment:
These `getattr` calls don't hit the database - they are just getting attributes off of they python object.  So no need to worry about performance, if that's what you were getting at.  I feel erring on the side of keeping the class object smaller in this case is better, unless really needed.